lib/public/components/UserAccount.js

```diff
@@ -364,8 +364,9 @@ class WatchedDeployment extends Component<Props & {target: string}> {
     if (projects) {
       for (var i = 0; i < projects.length; i++) {
         const p = projects[i]
-        if (!p.deployments) continue
-        const dp = p.deployments.find(dp => dp.id === target)
+        const dp = p.deployments
```

Comment:
Please use more descriptive variables. Please rename `p` and `dp` to more meaningful names.